src/lang-promql/promql.ts

```diff
@@ -57,6 +57,8 @@ export const promQLSyntax = LezerSyntax.define(
   }
 );
 
+const simplePromQLSyntax = LezerSyntax.define(parser);
```

Comment:
IMO the node props added above should always be added. They are necessary for correct highlighting, but they are separate from highlighting. I noticed the highlighter is already an external separate extension added in the example app (https://github.com/prometheus-community/codemirror-promql/blob/17cecae37707548e5bc7fb9906ef0d7f97e351e7/src/app/theme.ts#L6-L26).

When you were asking the question on chat, I thought you were considering moving that highlighter into the PromQL extension and making that optionally enabled.

Reply:
Well that's a way to deactivate the highlight :).

Once it's removed, then of course the highlighter is not working anymore. That's a bit the purpose. But if you activate it then you have the props added.

I know it's necessary for a correct highlightning, but if the purpose is to not highlight, then this is an (easy) solution to achieve, without altering the way codemirror is configured

Reply:
Ok, while that technically achieves an outcome where the user will not see highlighting, if the highlighter is still separate anyway, then it's IMO much better to just keep it that way and let the user control highlighting by either including a highlighter or not. I don't think we should be removing this node metadata...

Reply:
ok :) It wasn't a so good idea afterall :D. I will remove it then. I think I didn't get your point too in the tchat and didn't know it was already achievable with CMN natively.